src/Metadata/Extractor/ClassDescriptionExtractorInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Metadata\Extractor;
+
+/**
+ * Guesses the class's human readable description.
+ */
+interface ClassDescriptionExtractorInterface
```

Comment:
do we really need to introduce a new interface for this?

Reply:
I think it's better, like Symfony does: https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/PropertyInfo/PropertyDescriptionExtractorInterface.php.

Reply:
probably that we should put this code inside Symfony then ?

Reply:
Symfony has no component for it. `PropertyInfo` only cares about properties.